src/mode.js

```diff
@@ -59,15 +59,10 @@ function getMode_() {
       // occur during local dev.
       !!document.querySelector('script[src*="/dist/"],script[src*="/base/"]');
 
-  const overrideDevelopment = parseQueryString(location.hash)['development'];
-  const development = overrideDevelopment != undefined
-      ? overrideDevelopment == '1'
-      : !!document.querySelector('script[development]');
-
   return {
     localDev: isLocalDev,
     // Triggers validation
-    development: development,
+    development: !!parseQueryString(location.hash)['development'],
```

Comment:
This changes a bit the interpretation of `#development` from before. In particular, previously `#development=0` would be interpreted as `false`. Now this would yield `true`.